### PR TITLE
Adding conda_envfile

### PR DESCRIPTION
**Pull request by tdegeus**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by tdegeus** [comment]
@conda-forge/help-python ready for review

File: recipes/conda_envfile/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - conda_envfile_parse --version | grep {{ version }}
+
+about:
+  home: https://github.com/{{ org }}/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Simple command-line tools to clean / merge conda environment files.
+  description: Simple command-line tools to clean / merge conda environment files.
+  doc_url: https://github.com/{{ org }}/{{ name }}
```
Comment (jaimergp) [inline recipes/conda_envfile/meta.yaml:52]:
```suggestion
  doc_url: https://conda-envfile.readthedocs.io/
```

**Comment by tdegeus** [comment]
Thanks @jaimergp . I've updated

**Review by jaimergp — CHANGES_REQUESTED** [review]
Thanks! This is like a conda-less implementation of some parts of the `MatchSpec.merge` functionality. Nice!

There's an issue with the packaged files, though:

```
  adding 'conda_envfile/__init__.py'
  adding 'conda_envfile/_version.py'
  adding 'tests/__init__.py'
  adding 'tests/test_basic.py'
  adding 'conda_envfile-0.1.0.dist-info/LICENSE'
  adding 'conda_envfile-0.1.0.dist-info/METADATA'
  adding 'conda_envfile-0.1.0.dist-info/WHEEL'
  adding 'conda_envfile-0.1.0.dist-info/entry_points.txt'
  adding 'conda_envfile-0.1.0.dist-info/top_level.txt'
  adding 'conda_envfile-0.1.0.dist-info/RECORD'
  removing build/bdist.linux-x86_64/wheel
```

Your test files are being shipped as a global package, which is usually disregarded. You'll need to adjust the `exclude` options of the `find_packages` call, either in a patch or in a new (micro) release usptream.

Thanks again!

**Comment by tdegeus** [comment]
Thanks @jaimergp , I had not paid attention. Fixed it upstream.

**Comment by tdegeus** [comment]
@jaimergp Are you around to re-review?

**Comment by tdegeus** [comment]
Thanks!

**Pull request merged**